src/main/scala/strawman/collection/mutable/ListBuffer.scala

```diff
@@ -12,7 +12,7 @@ import scala.Predef.{assert, intWrapper}
 
 /** Concrete collection type: ListBuffer */
 class ListBuffer[A]
-  extends Seq[A]
+  extends GrowableSeq[A]
```

Comment:
I forgot that Arrays are not growable.  I guess there is a distinction there to maintain.